doc/source/ray-contribute/development.rst

```diff
@@ -71,35 +71,45 @@ Every time you want to update your local version you can pull the changes from t
 Prepare the Python environment
 ------------------------------
 
-.. tip::
+You probably want to have some type of Python virtual environment. For example, you could use Anaconda's ``conda``. 
 
-  To setup an isolated Anaconda environment, see :ref:`ray_anaconda`.
+.. tabbed:: conda
 
-You will probably want to have some type of Python virtual environment. For example you could use Python's integrated ``venv`` module to create one:
+    Set up an isolated Anaconda environment using the instructions in :ref:`ray_anaconda`.
```

Comment:
I think the referred instructions here are not super great for this particular use case -- they are instructions to install Ray from conda too, which is not typically what we want. It is better to just use conda for the base environment and then install ray with `pip`. I would recommend just having the instructions here and saying something like this

First you should set up a conda environment
```
conda create -c conda-forge python=3.9 -n ray
conda activate ray
```
and then you can install Ray with
```
pip install -U "ray[default]"
```

Reply:
Actually, I misspoke, since this is only about setting up the python environment and not about installing Ray (which will either happen automatically when the docs requirements are installed, or will happen by the instructions further below), let's drop the `pip install -U "ray[default]"` part :)

Reply:
ack